### PR TITLE
chore: add volunteer pranshuj73

### PR DESCRIPTION
**Pull request by pranshuj73**

### CODE REVIEW

File: data/users/pranshuj73.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Pranshu Jha",
+  "bio": "Full-Stack Developer",
+  "image": "https://avatars.githubusercontent.com/u/58560660?v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["Python", "JavaScript", "Web3"]
```
Comment (codewithvoid) [inline data/users/pranshuj73.json:8]: Add solidity here instead of web3?

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @pranshuj73. Added a small comment.

File: data/users/pranshuj73.json (hunk 1 above)
Comment (pranshuj73) [reply data/users/pranshuj73.json]: Changed web3 to solidity :]

**Comment by codewithvoid** [comment]
Thanks @pranshuj73 for making the changes. Welcome :)

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**